api/clients/automation/automation.go

```diff
@@ -202,19 +202,19 @@ func (a Client) Delete(ctx context.Context, resourceType ResourceType, id string
 }
 
 func (a Client) makeRequestWithAdminAccess(resourceType ResourceType, request func(options rest.RequestOptions) (*http.Response, error)) (*http.Response, error) {
```

Comment:
nit/later refactoring suggestion: as this method is called `makeRequestWithAdminAccess`, wouldn't it be cleaner to have the `if resourceType == Workflows` check outside before we call the method? If workflow -> withAdminAcces, else -> request()

As this won't functionally change the fix or the core libraries interfaces, fine to merge as is and follow up later